docs/source/framework/pytorch_integration/autograd_with_tc.rst

```diff
@@ -41,6 +41,13 @@ Examples
      out = convolution(I, W)
      out[0].sum().backward()
 
+.. note::
+
+    Please note the usage of :code:`.cuda()` i.e. tensor data is declared as the CUDA
+    type. Applying :code:`Variable` on the tensor data essentially is an extra no-op and
+    if :code:`Variable(torch.rand(), requires_grad=True).cuda()` is done, then the
+    grad will be available for the no-op and not the actual tensor data.
```

Comment:
I got lost here.  Why do we need that no-op in the first place?  And if it is a no-op, why does it matter if it receives grad or tensor data?

Reply:
however, the usage of Variable is going away in next pytorch release. So we have to live with it only few more days until next pytorch release happens

Reply:
That part is clear, mentioning no-op is not...